app/models/subject_exporter.rb

```diff
@@ -1,39 +1,74 @@
+require 'nokogiri'
+
```

Comment:
Would it make sense to move this class to `lib` instead of `app/models`?

Reply:
I was unsure about where to put this kind of code. `lib`, now that I see it, seems to make sense.